src/lib.rs

```diff
@@ -852,8 +852,8 @@ pub(crate) mod tests {
 
         fn size(&self) -> usize {
             let string_size = self.vstring.len();
-            let u32_size = std::mem::size_of::<u32>();
-            let bool_size = self.vbool.map_or(0, |_| std::mem::size_of::<bool>());
+            let u32_size = mem::size_of::<u32>();
```

Comment:
why to remove the `std` prefix

Reply:
Because this is not implemented in macro, this path can be removed (shown in gray in rustrover)

Reply:
make sense